samples/java/semantickernel-samples/pom.xml

```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
@SergeyMenshykh I think you added this Java sample. Is it related to the Java SK implementation?

Reply:
File removed since we are not going to alter files outside the java directory.